ansible/Dockerfile.CentOS6

```diff
@@ -2,20 +2,28 @@ FROM centos:6.9
 
 ARG user=jenkins
 
-RUN yum -y update; yum clean all
-RUN rpm -ivh http://dl.fedoraproject.org/pub/epel/6/i386/epel-release-6-8.noarch.rpm; \
- yum -y install ansible sudo; yum clean all
+# Install Python 3
+RUN yum -y update; yum clean all; \
+    yum -y install gcc openssl-devel bzip2-devel sqlite-devel sudo wget; \
+    cd /usr/src; \
+    wget -q https://www.python.org/ftp/python/3.6.10/Python-3.6.10.tgz; \
```

Comment:
seems a very specific and slightly older version of Python?

Reply:
This is the newest version I could get to compile on centos6